src/car_constants.py

```diff
@@ -8,5 +8,5 @@
 
 IMAGE_CROP_XMIN = 300
 IMAGE_CROP_XMAX = 1620
-IMAGE_CROP_YMIN = 520
-IMAGE_CROP_YMAX = 864
+IMAGE_CROP_YMIN = 570
+IMAGE_CROP_YMAX = 914
```

Comment:
These values are not universal. We use (520,864):
- in deployment: https://gitlab.cs.ut.ee/autonomous-driving-lab/e2e/e2e-workspace/-/blob/master/src/e2e_platform/nvidia_e2e/nvidia_e2e.py#L64-L65
- for EBM models training (will impact people's ability to replicate results in my paper): https://github.com/nikebless/ebm-driving/blob/master/data_extract/bag_extractor.py#L384

I saw your commit introducing 570 and 914, saying these work better. Ultimately we should then move the full stack to them, while this repo should maybe take them as an argument. What do you think?

Reply:
It is wrong in `nvidia_e2e.py` also, it is correct in `nvidia_waypoints.py`. If the model is trained using the pre-cropped dataset, it should use 570x914 as this was the crop used during dataset extraction. Models trained previously for the lidar vs camera paper used the old crop. Models in my thesis used the new crop, sadly I ran the steering model accidentally with the old crop and the waypoints node with the correct crop.

In hindsight would be a lot easier if I would not have changed the crop, these magic values are in like 3-5 repos now and hard to keep in sync. In the long run, it would be easier if everything used the same values. 

If the EBM models used the old crop values (520x864), then it makes sense to make these defaults and make the values I used in my thesis configurable.

Reply:
Actually, you are right, I did train all models in my thesis and the EBM project on the pre-cropped dataset, so the models are a bit out-of-distribution with the deployment, since there we use (520,864). 

I think for now it makes sense to keep 520,864, because this aligns with the deployment, and we use Vista to choose models that will perform well in deployment.

Reply:
Sure, rolled back crop changes.

Reply:
Can you clarify for me which directories on HPC contain which crops:
* `rally-estonia-cropped` (formerly `rally-estonia-cropped-new`)
* `rally-estonia-cropped-antialias`
* `rally-estonia-cropped-old` (formerly `rally-estonia-cropped`)

Do I understand correctly that we reverted to 520x864, which seems to be the most common over all codebases, but did not implement yet parameterization, so that you can choose other value at inference time? Am I correct that due to this some of the Romet's models can work non-optimally? Can we do the parameterization as well, or is it too much work to fit into current iteration?

Reply:
Yes, your understanding is correct. I'm over the time budget and don't have time to do it properly. Passing another set of parameters would be quite bad and some yaml based config would be more appropriate as Nikita suggested.